pacman-nodejs-app/docs/kubernetes-cluster-gke-federation.md

```diff
@@ -54,6 +54,17 @@ gcloud container clusters list
 export GCP_PROJECT=$(gcloud config list --format='value(core.project)')
 ```
 
+## Retrieve the Cluster credentials
+
+```
+gcloud config set container/use_client_certificate True
+export CLOUDSDK_CONTAINER_USE_CLIENT_CERTIFICATE=True
```

Comment:
Sorry I meant that with k8s 1.6 and without specifying either application_default_credentials or use_client_certificate, kubectl may be configured more seamlessly by gcloud without having to get-credentials. However, there may still be some RBAC stuff that has to happen as well.